main.js

```diff
@@ -194,18 +203,81 @@ function createWindow () {
   });
 }
 
+function createTrayIcon() {
+  logger.info('Creating tray icon');
+
+  // TODO: better to not use a hardcoded icon file
+  tray = new Tray(path.join(__dirname, '/images/icon_48.png'));
+
+  trayContextMenu = Menu.buildFromTemplate([
+      {id: 'show', label: locale.messages.show.message, visible: !mainWindow.isVisible(), click: () => {showWindow();}},
+      {id: 'hide', label: locale.messages.hide.message, visible: mainWindow.isVisible(), click: () => {hideWindow();}},
```

Comment:
I'm seeing a lot of `() => { something(); }` syntax in this PR, but that can be simplfied to `something`, no surrounding function required.

And in cases where it's something like `app.quit()` inside that function, we can change that to `app.quit.bind(app)`. That's a common pattern in this codebase.

Reply:
That was really silly of me, I moved the body of the lambdas to named functions, but I left the lambdas there...